pkg/get/get_test.go

```diff
@@ -6350,6 +6350,46 @@ func Test_DownloadOpenshiftCLI(t *testing.T) {
 	}
 }
 
+func Test_DownloadRosaCLI(t *testing.T) {
+	tools := MakeTools()
+	name := "rosa"
+
+	tool := getTool(name, tools)
+
+	const toolVersion = "1.2.23"
+
+	tests := []test{
+		{
+			os:      "linux",
```

Comment:
I think you are missing a few tests here for different archs. e.g. https://github.com/alexellis/arkade/blob/master/pkg/get/get_test.go#L6467